docs/framework/unmanaged-api/profiling/icorprofilercallback-initialize-method.md

```diff
@@ -26,10 +26,12 @@ HRESULT Initialize(
     [in] IUnknown     *pICorProfilerInfoUnk);  
 ```  
   
-## Parameters  
- `pICorProfilerInfoUnk`  
- [in](/cpp/atl/iunknown) interface that the profiler must query for an [ICorProfilerInfo](../../../../docs/framework/unmanaged-api/profiling/icorprofilerinfo-interface.md) interface pointer.  
-  
+## Parameters
+
+- `pICorProfilerInfoUnk`
+
+  [in](/cpp/atl/iunknown) interface that the profiler must query for an [ICorProfilerInfo](../../../../docs/framework/unmanaged-api/profiling/icorprofilerinfo-interface.md) interface pointer.  
```

Comment:
```suggestion
  \[in](/cpp/atl/iunknown) interface that the profiler must query for an [ICorProfilerInfo](../../../../docs/framework/unmanaged-api/profiling/icorprofilerinfo-interface.md) interface pointer.  
```

Reply:
@mairaw This one shouldn't be escaped as it shows a link.

Reply:
Actually, I found this was an error introduced a while back. 
https://github.com/dotnet/docs/commit/835994638c63e50a25821b7af0c9892b1e442fee#diff-e004b80f303a144715b1d4d9aa8d25f7

This should be the appropriate change:
```suggestion
  \[in] Pointer to an [IUnknown](/cpp/atl/iunknown) interface that the profiler must query for an [ICorProfilerInfo](../../../../docs/framework/unmanaged-api/profiling/icorprofilerinfo-interface.md) interface pointer.  
```

Reply:
Ugh, looking at that commit I see several issues like that introduced.

Reply:
I've fixed it for this particular line. I'll open an issue for the rest of these issues and fix them in a separate PR. All the errors in that commit are related to the in and out hyperlinks added in wrong place right ?

Reply:
Correct. Probably the script was using some regex to find the start of a link with [ and it messed up some of these links.